torch/fft/__init__.py

```diff
@@ -87,6 +87,98 @@
     tensor([0.+0.j, 1.+0.j, 2.+0.j, 3.+0.j])
 """)
 
+fftn = _add_docstr(_fft.fft_fftn, r"""
+fftn(input, s=None, dim=None, norm=None) -> Tensor
+
+Computes the N dimensional discrete Fourier transform of :attr:`input`.
+
+Note:
+
+    The Fourier domain representation of any real signal satisfies the
+    Hermitian property: ``X[i_1, ..., i_n] = conj(X[-i_1, ..., -i_n])``. This
+    function always returns all positive and negative frequency terms, even
+    though for real inputs, half of these values are redundant.
+    :func:`~torch.fft.rfftn` returns the more efficient "half-complex"
```

Comment:
"efficient" is vague. What about "returns the more compact..."

I know we discussed this briefly on the last PR, but is "half-complex" really the term of art for this?

Reply:
If there's not a good term of art, do we need to name this?

"returns a more compact representation containing only the positive frequencies of the last dimension."

Reply:
This wording comes from the `rfft` docs. I must have missed it when I changed "half-complex" to "one-sided" elsewhere. I've updated both PRs to use "one-sided".